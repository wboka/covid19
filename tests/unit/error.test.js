import { mount, RouterLinkStub } from '@vue/test-utils'
import Error from '@/layouts/error.vue'

describe('Error Page', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Error, {
      propsData: {
        error: {
          statusCode: 404
        }
      },
      stubs: {
        "NuxtLink": RouterLinkStub
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
