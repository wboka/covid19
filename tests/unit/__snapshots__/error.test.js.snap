// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page is a Vue instance 1`] = `
<div
  class="text-center"
>
  <h2>
    Page not found
  </h2>
   
  <p
    class="text-center"
  >
    <a>
      Home page
    </a>
  </p>
</div>
`;
